packages/client/components/transaction-details/TransactionDetails.tsx

```diff
@@ -258,7 +261,9 @@ export function TransactionDetails({ txn, wallet, navigation, otherDetails }: Pr
             />
             <MultiLineListItem
                 label={"Fee"}
-                value={`${getFormattedAmount(String(0), wallet.currencyType)} ${typeToISOCode[wallet.currencyType]}`}
+                value={`${getFormattedAmount(String(tranasctionFee), wallet.currencyType)} ${
```

Comment:
I know its not your code but those type casting functions are no good

```suggestion
                value={`${getFormattedAmount(tranasctionFee.toString(), wallet.currencyType)} ${
```